### PR TITLE
fix: outdated types

### PR DESCRIPTION
**Pull request by eeston**
Fix for https://github.com/malijs/mali/issues/235

### CODE REVIEW

**Review by anonrig — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -62,11 +62,11 @@
     "traverse": "^0.6.6"
   },
   "peerDependencies": {
-    "@grpc/grpc-js": "^1.2.0",
```
Comment (bojand) [inline package.json:65]: can we leave this to `^1.2.0` please. There is no reason to strictly require a specific patch version of core grpc package.

Comment (eeston) [reply package.json]: No problem  👍

**Comment by eeston** [comment]
I've addressed your comment @bojand. Let me know if you're happy and I'll squash the commits for you to merge 👍

**Review by bojand — APPROVED** [review]

**Comment by eeston** [comment]
Squashed the commit. Thanks guys

**Comment by anonrig** [comment]
Hey @eeston, thank you for your contribution. I'll be releasing 0.41.0 asap. Here's the release tag: https://github.com/malijs/mali/releases/tag/0.41.0

**Pull request merged**